lib/src/models/album.dart

```diff
@@ -97,4 +97,12 @@ class AlbumSimple extends Object {
 
   /// The Spotify URI for the album.
   String uri;
+
+  /// The tracks of this album wrapped in a page object.
+  @JsonKey(name: 'tracks')
+  Paging<TrackSimple> tracksPage;
+
+  Iterable<TrackSimple> get tracksNative =>
+      tracksPage?.itemsNative?.map((e) => TrackSimple.fromJson(e))?.toList() ??
+      [];

```

Comment:
In a second thought, maybe the right place for something like this is in src/endpoints/tracks.dart using  `Pages.fromPaging`
Something like
`src/endpoints/tracks.dart:`
```dart
Pages<TrackSimple> fromPaging(Paging<TrackSimple> paging) => 
    Pages<TrackSimple>.fromPaging(_api, paging, (json) => TrackSimple.fromJson(json));

```

Reply:
This is a good idea. I'll see what I can do. Thank you!